pkgs/development/ocaml-modules/sel/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, lib, fetchurl, ocamlPackages, buildDunePackage }:
+
+buildDunePackage rec {
+  pname = "sel";
+  version = "0.4.0";
+
+  src = fetchurl {
+    url = "https://github.com/gares/sel/releases/download/v${version}/sel-${version}.tbz";
+    hash = "sha256-Sp0eSeKyFmMvOQrebqLKP+HbHQB0D0eAmEjDzxnFL1I=";
+  };
+
+  buildInputs = with ocamlPackages; [
+    ppxlib
+    ppx_deriving
+  ];
+
+  meta = {
+    description = "A simple event library";
+    homepage = "https://github.com/gares/sel/";
+    license = lib.licenses.mit;
+    maintainers = [ lib.maintainers.maximedenes ];
+    platforms = lib.platforms.all;
```

Comment:
FYI, `buildDunePackage` sets a default value to `meta.platforms`, unless explicitly overridden as done here. The default value is `ocaml.meta.platforms` so is probably more accurate than `all`.